liquibase-service/src/main/resources/db/rdb_modern/routines/018-sp_d_lab_test_postprocessing-001.sql

```diff
@@ -0,0 +1,2102 @@
+	
+		    INSERT INTO [dbo].[job_flow_log]
+                (batch_id,[Dataflow_Name],[package_Name] ,[Status_Type],[step_number],[step_name],[row_count])
+                VALUES(@batch_id,'D_LAB_TEST','D_LAB_TEST','START',@Proc_Step_no,@Proc_Step_Name,@RowCount_no);
+
+
+			COMMIT TRANSACTION;
+
+            BEGIN TRANSACTION; 
+			SET @PROC_STEP_NO =  @PROC_STEP_NO + 1 ;
+			SET @PROC_STEP_NAME = ' GENERATING #LAB_TEST1_TMP '; 
+
+
+			  IF OBJECT_ID('#LAB_TEST1_TMP', 'U') IS NOT NULL 
+			         drop table  #LAB_TEST1_TMP ;
+
+ 		
+							   select  lt1.*,lto.*,ltmi.*,lti.*, lti.Lab_Rpt_Uid as Lab_Rpt_Uid_Test1
+									into  #LAB_TEST1_TMP
+ 									from  #LAB_TEST1_uid lt1
+ 									  left outer join  #LAB_TEST_oth  lto on lt1.LAB_TEST_uid = lto.LAB_TEST_uid_oth
+									  left outer join  #LAB_TEST_mat_init  ltmi  on lt1.LAB_TEST_uid = ltmi.LAB_TEST_uid_mat
+									  left outer join  #LAB_TESTinit  lti on lt1.LAB_TEST_uid = lti.LAB_TEST_uid_Test
+								;
+
+				ALTER TABLE #LAB_TEST1_TMP DROP COLUMN Lab_Rpt_Uid;
+
+	  
+
+
+			SELECT @RowCount_no = @@ROWCOUNT;
+	
+		    INSERT INTO [dbo].[job_flow_log]
+                (batch_id,[Dataflow_Name],[package_Name] ,[Status_Type],[step_number],[step_name],[row_count])
+                VALUES(@batch_id,'D_LAB_TEST','D_LAB_TEST','START',@Proc_Step_no,@Proc_Step_Name,@RowCount_no);
+
+
+			COMMIT TRANSACTION;
+
+            BEGIN TRANSACTION; 
+			SET @PROC_STEP_NO =  @PROC_STEP_NO + 1 ;
+			SET @PROC_STEP_NAME = ' GENERATING #LabReportMorb '; 
+
+
+			  IF OBJECT_ID('#LabReportMorb', 'U') IS NOT NULL 
+			         drop table  #LabReportMorb;
+
+
+						  select * 
+						  into #LabReportMorb
+						  from #LAB_TEST1_TMP
+						 where  LAB_TEST_type in ('Order', 'Result', 'Order_rslt') and  oid =4
+						;
+
+			SELECT @RowCount_no = @@ROWCOUNT;
+	
+		    INSERT INTO [dbo].[job_flow_log]
+                (batch_id,[Dataflow_Name],[package_Name] ,[Status_Type],[step_number],[step_name],[row_count])
+                VALUES(@batch_id,'D_LAB_TEST','D_LAB_TEST','START',@Proc_Step_no,@Proc_Step_Name,@RowCount_no);
+
+
+			COMMIT TRANSACTION;
+
+
+			-- IMPORTANT  - ASK UPASANA ABOUT THIS LOGIC
+            BEGIN TRANSACTION; 
+			SET @PROC_STEP_NO =  @PROC_STEP_NO + 1 ;
+			SET @PROC_STEP_NAME = ' GENERATING #Morb_OID '; 
+
+
+			  IF OBJECT_ID('#Morb_OID', 'U') IS NOT NULL 
+			         drop table  #Morb_OID ;
+
+								select l.*, 
+								o.PROGRAM_JURISDICTION_OID as Morb_oid,
+								l.Lab_Rpt_Uid_Test1 as Lab_Rpt_Uid_Mor,
+								l.LAB_TEST_uid as LAB_TEST_uid_mor ,
+								l.LAB_TEST_uid_oth as LAB_TEST_uid_oth_mor
+								into  #Morb_OID
+								from #LabReportMorb l,
+									dbo.nrt_observation l_extension,
+									 dbo.nrt_observation o
+								where 
+								l.Lab_Rpt_Uid_Test1 = l_extension.observation_uid
+								and o.observation_uid = l_extension.report_observation_uid -- column seems to be missing, also is it even the right column
```

Comment:
This should have been addressed in one of the Observation service fixes. Is the issue still persisting?